src/components/ReportActionItem/IOUQuote.js

```diff
@@ -1,46 +1,48 @@
 import React from 'react';
-import {View} from 'react-native';
+import {View, Pressable} from 'react-native';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
 import Str from 'expensify-common/lib/str';
 import Text from '../Text';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
 import styles from '../../styles/styles';
+import themeColors from '../../styles/themes/default';
 import reportActionPropTypes from '../../pages/home/report/reportActionPropTypes';
 import withLocalize, {withLocalizePropTypes} from '../withLocalize';
 
 const propTypes = {
     /** All the data of the action */
     action: PropTypes.shape(reportActionPropTypes).isRequired,
 
-    /** Should the View Details link be displayed? */
-    shouldShowViewDetailsLink: PropTypes.bool,
-
     /** Callback invoked when View Details is pressed */
     onViewDetailsPressed: PropTypes.func,
 
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
-    shouldShowViewDetailsLink: false,
     onViewDetailsPressed: () => {},
 };
 
 const IOUQuote = props => (
     <View style={[styles.chatItemMessage]}>
         {_.map(props.action.message, (fragment, index) => (
-            <View key={`iouQuote-${props.action.reportActionID}-${index}`} style={[styles.alignItemsStart, styles.blockquote]}>
-                <Text style={[styles.chatItemMessage]}>
-                    {Str.htmlDecode(fragment.text)}
-                </Text>
-                {props.shouldShowViewDetailsLink && (
+            <View key={`iouQuote-${props.action.reportActionID}-${index}`} style={[styles.textInputAndIconContainer, styles.alignItemsStart, styles.justifyContentBetween]}>
+                <Text>
                     <Text
-                        style={[styles.chatItemMessageLink, styles.alignSelfStart]}
+                        style={[styles.chatItemMessageLink]}
                         onPress={props.onViewDetailsPressed}
                     >
-                        {props.translate('iou.viewDetails')}
+                        {Str.htmlDecode(fragment.text.split(' ')[0])}
```

Comment:
When I first saw this, I thought this would cause breaking behavior with translations (i.e. not translating). But then I tested it, and it totally works! So nice catch! Do you happen to know why this works? I'm thinking the message must get translated upstream, but I went as far as ReportActionItem and didn't see the call to `translate` anywhere. 

Anyway, now that we don't need this method here, we can also remove the import for withLocalize, the withLocalize props, and just export the component with `export default IOUQuote` instead of passing it to withLocalize. 

Nice catch!

Reply:
https://github.com/Expensify/App/blob/main/src/libs/ReportUtils.js#L703-L729
This is where the fragment is coming from! Which brings up a great point- this isn't using translate. I think I can fix that  here as well?

Reply:
<img width="1635" alt="image" src="https://user-images.githubusercontent.com/38015950/202818260-aa849171-5f60-4576-baec-ecf6a6c5c8ed.png">
In spanish ^^

Reply:
Commenting this for reference later, I think since these are considered a report action, it doesn't update with localize? I can get it to work on send, but not on page load, refresh, or switching languages- it always reverts to english. For now, I'm going to ignore since the method you pointed out would still work if we switched languages

in en.js iou:
``` js
        requestedAction: ({amount, comment, who}) => `Requested ${amount} from ${who}${comment && ` for ${comment}`}`,
        splitAction: ({amount, comment, who}) => `Split ${amount} with ${who}${comment && ` for ${comment}`}`,
        cancelledAction: ({amount, comment}) => `Cancelled the ${amount} request${comment && ` for ${comment}`}`,
        declinedAction: ({amount, comment}) => `Declined the ${amount} request${comment && ` for ${comment}`}`,
        paidAction: ({amount, comment, who}) => `Paid ${amount} to ${who}${comment && ` for ${comment}`}`,
```

in es.js iou:
``` js
        requestedAction: ({amount, comment, who}) => `Solicitó ${amount} de ${who}${comment && ` por ${comment}`}`,
        splitAction: ({amount, comment, who}) => `Split ${amount} con ${who}${comment && ` por ${comment}`}`,
        cancelledAction: ({amount, comment}) => `Canceló la solicitud de ${amount} ${comment && ` por ${comment}`}`,
        declinedAction: ({amount, comment}) => `Se rechazó la solicitud de ${amount} ${comment && ` por ${comment}`}`,
        paidAction: ({amount, comment, who}) => `Pagué ${amount} a ${who}${comment && ` por ${comment}`}`,
```

in reportUtils getIOUReportActionMessage()
``` js
    let iouMessage;
    switch (type) {
        case CONST.IOU.REPORT_ACTION_TYPE.CREATE:
            iouMessage = Localize.translateLocal('iou.requestedAction', {amount, who, comment});
            break;
        case CONST.IOU.REPORT_ACTION_TYPE.SPLIT:
            iouMessage = Localize.translateLocal('iou.splitAction', {amount, who});
            break;q
        case CONST.IOU.REPORT_ACTION_TYPE.CANCEL:
            iouMessage = Localize.translateLocal('iou.cancelledAction', {amount, who, comment});
            break;
        case CONST.IOU.REPORT_ACTION_TYPE.DECLINE:
            iouMessage = Localize.translateLocal('iou.declinedAction', {amount, who});
            break;
        case CONST.IOU.REPORT_ACTION_TYPE.PAY:
            iouMessage = Localize.translateLocal('iou.paidAction', {amount, who, comment});
            break;
        default:
            break;
    }
```